packages/switchToggle/src/Default/Default.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { Form, ThemeProvider } from 'react-bootstrap';
+import { SwitchToggleDefaultProps } from './Default.types';
+import * as styleMod from './styles.module.scss';
+import { createLocalProps } from '@owlui/utils';
+
+export const Component = (props: SwitchToggleDefaultProps) => {
+  const baseClass = 'switchToggle';
+  const prefix = props.prefix || '';
+
+  const locals: SwitchToggleDefaultProps = createLocalProps(
+    props,
+    {
+      module: styleMod,
+      classes: {
+        base: baseClass,
+        prefix: prefix,
+        optionals: [
+          {
+            fields: ['theme'],
+            value: 'theme',
+          },
+          {
+            fields: ['appearance'],
+            value: 'Appearance',
+          },
+          {
+            fields: ['size'],
+            value: 'Size',
+          },
+        ],
+      },
+    },
+    ['prefix', 'theme', 'appearance', 'Size']
+  );
+
+  return (
+    <>
+      <ThemeProvider
+        prefixes={{
+          'form-check': 'owlui-form-check',
+          'form-switch': 'owlui-form-switch',
+          'form-check-input': 'owlui-form-check-input',
+          label: 'owlui-label',
+        }}
+      >
+        <Form.Check
+          {...locals}
+          // onClick={() => setToggle(!toggle)}
```

Comment:
Check out and clean up all the component files looking for comments that are not describing or clarifying something.